__tests__/app.test.js

```diff
@@ -274,3 +272,14 @@ describe.only("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("DELETE api/comments/:comment_id", () => {
+  it('Status 204 "No Content" - Delete Comment', () => {
+    return request(app).delete("/api/comments/5").expect(204);
+  });
+  it('Status 404 "Not Found" - Comment id does not exists', () => {
+    return request(app).delete("/api/comments/1123423").expect(404);
+  });
+  it('Status 400 "Bad Request" - Incorrect comment id', () => {
+    return request(app).delete("/api/comments/ada3q3e").expect(400);
```

Comment:
nice, for your errors I'd just check you're getting back the message you expect.